package.json

```diff
@@ -2,6 +2,7 @@
   "name": "lighthouse",
   "version": "1.0.0",
   "description": "",
+  "main": "./src/module.js",
```

Comment:
any reason not to have this under ./module/index.js? Just so that it matches the cli entry point and keeps src to the core library.